platform/src/main/java/org/hillview/storage/CassandraSSTableLoader.java

```diff
@@ -171,17 +235,78 @@ public Schema getSchema(CFMetaData metadata) {
         }
     }
 
+    public Schema getSchema() {
+        return this.actualSchema;
+    }
+
+    private static SSTableReadsListener newReadCountUpdater() {
+        return new SSTableReadsListener() {
+            @Override
+            public void onScanningStarted(SSTableReader sstable) {
+                // No need to do sstable.incrementReadCount();
+            }
+        };
+    }
+
+    private void computeRowCount() {
+        this.rowCount = 0;
+        PartitionColumns newColDef = this.columnDefinitions;
+        for (ColumnDefinition columnDefinition : this.columnDefinitions)
+            newColDef = newColDef.without(columnDefinition);
```

Comment:
There must be a way to create empty column definitions instead of adding all and removing them.